src/main/java/org/folio/service/SynchronizationJobService.java

```diff
@@ -161,8 +161,8 @@ private Future<SynchronizationJob> updateJobStatus(SynchronizationJob job,
     SynchronizationStatus syncStatus) {
 
     return syncRepository.update(job.withStatus(syncStatus.getValue()))
-      .onSuccess(r -> log.info("Synchronization job status updated: " + syncStatus.getValue()))
-      .onFailure(t -> log.error("Failed to update synchronization job status: " + t.getMessage()))
+      .onSuccess(r -> log.info("Synchronization job status updated: {}", syncStatus.getValue()))
+      .onFailure(t -> log.error("Failed to update synchronization job status: {}", t.getMessage()))
```

Comment:
Should we also pass throwable to log.error in order to log it properly?

Reply:
done